packages/core/src/xml-transformations/xslt-vocabulary.ts

```diff
@@ -0,0 +1,7 @@
+export const XSLT_LIFTING = {
+  Generator: "http://example.com/generator/xslt-lifting",
```

Comment:
@jakubklimek  We should not be using example namespace any longer, also now that we have the demo we should have our own prefix right?